utils/markdown/remove_markdown.js

```diff
@@ -5,15 +5,15 @@ import marked from 'marked';
 
 export default class RemoveMarkdown extends marked.Renderer {
     code(text) {
-        return ' ' + text.split('\n').join(' ');
+        return text.split('\n').join(' ');
```

Comment:
These split/joins could be simplified down to `test.replace(/\n/g, ' ');`. That might also be a bit faster